src/query/query.pest

```diff
@@ -11,7 +11,7 @@ funcNames          = _{ latest }
 latest             =  { "latest" }
 
 infixExpression = { lookup ~ infixFunction ~ lookupValue }
-infixFunction   = { ("=" | "!" | "<" | ">"){2} }
+infixFunction   = @{ ("=" | "!" | "<" | ">"){1,2} }
```

Comment:
Can now be length 1 with `<` or `>`

Reply:
Would this also accept 2 charcter operators that aren't supported, like ">!"? ..Oh, looks like you get an error if you provide an invalid operator.

Reply:
i think if we are too strict with the infix function at the pest stage then we lose the ability to give more meaningful error messages, lets say someone wrote `=` instead of `==` or something like that, pest would throw its own error but if we allow it through pest, we could say "did you mean ==". I think keeping the infixFunction rule intentionally lax is probably a good thing